packages/solid/web/src/server-mock.ts

```diff
@@ -1,19 +1,29 @@
 //@ts-nocheck
+function throwInBrowser(func: Function) {
+  const err = new Error(`${func.name} is not supported in the browser, returning undefined`);
+
+  console.error(err);
```

Comment:
This is debatable, we can try throwing, but sometimes may the user's making something env agnostic, so they don't want the app breaks while importing/using such functions!

Reply:
I'm ok with the warning. Knowing what is wrong is the most important.